pensjon-brevbaker-api-model/src/main/kotlin/no/nav/pensjon/brev/api/model/vedlegg/OpplysningerBruktIBeregning.kt

```diff
@@ -80,6 +94,10 @@ data class OpplysningerBruktIBeregningUTDto(
         val tellerTTEOS: Int?,
         val tellerTTNordiskKonv: Int?,
         val utenforEOSogNorden: UtenforEOSogNorden?,
+        val framtidigTTEOS: Int?,
+        val fastsattTrygdetid: Int?,  //  TODO: Beregnes i Exstream: (<FaTTNorge> + <FramtidigTTNorge>)/12
+        val harTrygdetidsgrunnlag: Boolean,  // TODO: If a trygdeperiod exists, true
```

Comment:
harTrygdetidsgrunnlag kan erstattes med å sjekke om minst en av listene med trygdetidsperioder for brukeren sin størrelse er større enn 0

Reply:
TODO: TrygdetidenDin.kt